vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -695,8 +709,7 @@ class MessageItemFactory @Inject constructor(
         return this
                 ?.filterNotNull()
                 ?.map {
-                    // Value comes from AudioRecordView.maxReportableAmp, and 1024 is the max value in the Matrix spec
```

Comment:
This comment is still valuable, no? Maybe just update the part about `AudioRecordView.maxReportableAmp` to `AudioWaveformView.MAX_FFT`

Reply:
Right, done.